### ISSUE TITLE
Latest update not syncing account devices or with discord rich presence.

### ISSUE THREAD

**Issue by E85Addict** [SpotX-Official/SpotX#141]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://github.com/amd64fox/SpotX/discussions/111#discussion-4304353">described in the FAQ</a>

### Сountry of your account

US

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: 1.1.94.864.g540a4cc6
- Windows: Windows 11 21H2 22000.978
- PowerShell: 7.2.6
```

### 📝 Description

1.Update to latest Spotify using SpotX
2.Listen to any song
3. Discord doesn't show a status and spotify on phone doesn't show its playing a song on the same account where the previous version 1.1.93.896.g3ae3b4f3 did. It would show that the PC was listening to something and discord would also display it, with latest update the app is unaware of pc playback and it reports no music to discord I can listen to different music on both devices and instead of discord picking up the pc playing its music it picks up the phone almost like the new update has secluded itself or maybe blocked itself from broadcasting the listening music information.
...

**Comment by amd64fox** [comment]
Who has a similar problem, roll back to [version 93](https://cutt.ly/8EH6NuH) and block updates

Let's hope the developers fix this soon..

**Comment by E85Addict** [comment]
> Who has a similar problem, roll back to [version 93](https://cutt.ly/8EH6NuH) and block updates
> 
> Let's hope the developers fix this soon..

There is a error when using the previous version, and lyrics don't work I will attach screenshot
![image](https://user-images.githubusercontent.com/77761710/190096466-c1b29c78-d5f1-4592-959d-5110d554c1b5.png)

**Comment by amd64fox** [comment]
> There is a error when using the previous version, and lyrics don't work

fixed, update spotx

**Comment by amd64fox** [comment]
version bump to `1.1.94.870`
spotify connect still not working well

> **Note**
i created a post in [community.spotify.com](https://community.spotify.com/t5/Desktop-Windows/Spotify-Connect-not-working/td-p/5429730), support with likes and comments so that the Spotfiy developers fix it faster

**Comment by E85Addict** [comment]
Is spotify connect the issue at hand?

**Comment by amd64fox** [comment]
The bug was fixed in the new version `1.1.94.872`, update spotx
